.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+​
+- [ ] Changelog entry have been added / updated
```

Comment:
what if no changelog entry is needed? will we still check this, or remove the line?

Reply:
```suggestion
- [ ] Changelog entry have been added / updated / is not needed
```

Reply:
do we have to add `have been added` at all?